cmd/forkooor.js

```diff
@@ -2471,6 +2752,25 @@ const createCompV3Position = async (
             process.exit(0);
         });
 
+    program
+        .command('get-morphoAaveV2-position <isEOA> [addr]')
+        .description('MorphoAaveV2 position view, default to proxy')
+        .action(async (isEOA, addr) => {
+            const { senderAcc, proxy } = await forkSetup(addr);
+            const view = await getContractFromRegistry('MorphoAaveV2View', undefined, false, true);
+
+            // compareAddr is just a string compare
+            const user = compareAddr(isEOA, 'false') ? proxy.address : senderAcc.address;
```

Comment:
Rename the helper function `compareAddr` if using for other purpose and has no specific ties to eth address